crates/hotshot-stake-table/src/vec_based.rs

```diff
@@ -0,0 +1,455 @@
+//! A vector based stake table implementation. The commitment is the rescue hash of the list of (key, amount) pairs;
```

Comment:
For this hash-based implementation, are we ever going to use any update/lookup APIs?

Reply:
We have update and lookup, w/o proof.